<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/london/buses.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>20</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame, QLabel, QToolTip {
	background-color: rgb(0, 255, 0,0);
    border: 0px solid green;
	font: 17pt &quot;Rockwell&quot;;
    border-radius: 7px;
    padding: 2px;
   
}</string>
   </property>
   <property name="text">
    <string>Please enter your bus and his stop with time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>487</width>
     <height>217</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/prefix/img/just_bus.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEone">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>90</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>23</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	font: 87 10pt &quot;Arial Black&quot;;
    border: 2px solid black;
    border-radius: 10px;
    padding: 0 8px;
    background: ;
    
	background-color: rgb(0, 255,0, 0);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>240</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushBone">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	
	background-color: rgb(0, 255,0,0);
    border:2px solid #c50000;
    border-radius: 5px;
   
  
	
    min-width: 80px;
	
	color: rgb(200, 0, 0);
	
	font: 9pt &quot;Rockwell&quot;;
	
}

QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f9781c, stop: 1 #c6460b);
 
}

QPushButton:flat {
      border: none; /* no border for a flat push button */
 
}

QPushButton:default {
      border:2px solid #000000  ; /* make the default button prominent */
	
	

}</string>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushBtwo">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	
	background-color: rgb(0, 255,0,0);
    border:2px solid #c50000;
    border-radius: 5px;
   
  
	
    min-width: 80px;
	
	color: rgb(200, 0, 0);
	
	font: 9pt &quot;Rockwell&quot;;
	
}

QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f9781c, stop: 1 #c6460b);
 
}

QPushButton:flat {
      border: none; /* no border for a flat push button */
 
}

QPushButton:default {
      border:2px solid #000000  ; /* make the default button prominent */
	
	

}</string>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
